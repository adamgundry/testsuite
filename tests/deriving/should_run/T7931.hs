{-# LANGUAGE StandaloneDeriving #-} 
module Main where 
 
data A 
deriving instance Read A 
 
main = seq (read "" :: A) (return ()) 
