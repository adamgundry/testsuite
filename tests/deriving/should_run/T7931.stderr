T7931: Derived Read on empty data type
